bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileOrRendition.java

```diff
@@ -248,7 +249,6 @@ private HttpResponse initiateDownload() throws IOException {
                     connection = clientProvider.getHttpClientSupplier().get().execute(lastRequest);
                     size = connection.getEntity().getContentLength();
                 } catch (IOException | IllegalArgumentException ex) {
-                    Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
We should log this properly too, I think.  Errors in this class don't always bubble up where they can be inspected easily.